### PR TITLE
move python lcmtypes back into `drake` package to work around lcm bug

### PR DESCRIPTION
**Pull request by rdeits**
I gave up on using the `drake` package for the python bindings, so we no longer need to suffer because of https://github.com/RobotLocomotion/drake/issues/1425 This moves the package back to just being called `drake`, and should allow us to move past lcm 1.2

Corresponding pull requests: 

* https://github.com/RobotLocomotion/cmake/pull/31
* https://github.com/RobotLocomotion/director/pull/167

Once we have passing tests here, we can merge those two other PRs and then switch the git URLs to point to the RobotLocomotion repos for cmake and director.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -102,13 +102,13 @@ set(bullet_GIT_REPOSITORY https://github.com/RobotLocomotion/bullet-pod.git)
...
-set(cmake_GIT_REPOSITORY https://github.com/RobotLocomotion/cmake.git)
-set(cmake_GIT_TAG 2d8eebdaf3a7e7a7125c1190462bc02fb3a83824)
+set(cmake_GIT_REPOSITORY https://github.com/rdeits/cmake.git)
+set(cmake_GIT_TAG 800fecfa7842f918d96a96606fa61eb99ce4d03f)
 set(cmake_GIT_CLONE_DIR "${PROJECT_SOURCE_DIR}/drake/cmake")
 set(cmake_NO_BUILD TRUE)
 set(cmake_IS_PUBLIC TRUE)
-set(director_GIT_REPOSITORY https://github.com/RobotLocomotion/director.git)
-set(director_GIT_TAG 628669589d589577dbc1f372529bab8a1e75c926)
+set(director_GIT_REPOSITORY https://github.com/rdeits/director.git)
```
Comment (patmarion) [inline CMakeLists.txt]: you shouldn't need to change the url for director_GIT_REPOSITORY.

Comment (rdeits) [inline CMakeLists.txt]: yup, now that https://github.com/RobotLocomotion/director/pull/167 is merged I'll restore it

**Comment by rdeits** [comment]
(this is now a bit more pressing that it was originally, since our current version of LCM is broken with gcc 4.9: #1663 )

File: CMakeLists.txt [hunk 2]
```diff
@@ -102,13 +102,13 @@ set(bullet_GIT_REPOSITORY https://github.com/RobotLocomotion/bullet-pod.git)
 set(bullet_GIT_TAG 4319ffd7e9251066d93064f5a8dab12e33dbe5e2)
 set(bullet_IS_PUBLIC TRUE)
 set(bullet_IS_CMAKE_POD TRUE)
-set(cmake_GIT_REPOSITORY https://github.com/RobotLocomotion/cmake.git)
-set(cmake_GIT_TAG 2d8eebdaf3a7e7a7125c1190462bc02fb3a83824)
+set(cmake_GIT_REPOSITORY https://github.com/rdeits/cmake.git)
```
Comment (patmarion) [inline CMakeLists.txt]: does this url need to be reverted, too?

**Comment by patmarion** [comment]
Agreed.  After this pull request, we can discuss upgrading to the latest LCM release.

File: CMakeLists.txt (hunk 2 above)
Comment (rdeits) [inline CMakeLists.txt]: ah, yes, forgot about that one. Thanks!

**Pull request merged**